apps/web/src/app/page.tsx

```diff
@@ -1,14 +1,20 @@
-import { listCommits } from '@latitude-data/core'
+import { Commit, listCommits } from '@latitude-data/core'
+import { exampleEnqueue } from '$/actions/example-enqueu'
 import database from '$/db/database'
 
+export const dynamic = 'force-dynamic'
```

Comment:
what's this

Reply:
Next try to build the pages when doing `pnpm build`. This makes everything fail because we don't have env variables at build time

Reply:
No, the error is that the variables are not defined and we can't connect to DB or Redis at build time. 

All good. I want to avoid these errors if possible. But the build succeed anyway